schemas/Clinical/index.js

```diff
@@ -76,6 +76,40 @@ const typeDefs = gql`
     donorId: String!
     specimenId: String!
   }
+  
+  type ClinicalSubmissionData @cost(complexity: 10) {
+    id: ID    
+    state: String
```

Comment:
If there is a known list of possible states, lets use an enum with those values.

Reply:
there is, I will add it then.